src/main/java/com/smartcar/sdk/AuthClient.java

```diff
@@ -77,11 +78,10 @@ public Builder redirectUri(String redirectUri) {
       return this;
     }
 
-    public Builder testMode(boolean testMode) {
-      this.testMode = testMode;
+    public Builder mode(String mode) {
+      this.mode = mode;
       return this;
     }
-
     public AuthClient build() throws Exception {
       if (this.clientId == null) {
```

Comment:
Can you add validation of the mode parameter in this method (same check that you did in the other SDKs)